src/main/kotlin/org/rust/ide/refactoring/introduceVariable/impl.kt

```diff
@@ -35,7 +35,7 @@ private class ExpressionReplacer(
     private val psiFactory = RsPsiFactory(project)
     private val suggestedNames = chosenExpr.suggestedNames()
 
-    fun replaceElementForAllExpr(exprs: List<RsExpr>, postfixLet: Boolean) {
+    fun replaceElementForAllExpr(exprs: List<RsExpr>, postfixLet: Boolean, commandName: String, groupId: String) {
```

Comment:
Let's add `@NlsContexts.Command` here as well since the corresponding inspection doesn't provide inter function analysis, and you have to pass it manually